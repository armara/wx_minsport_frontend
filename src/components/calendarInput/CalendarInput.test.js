import React from 'react'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer'

import DayPickerInput from 'components/calendarInput/CalendarInput'

configure({ adapter: new Adapter() })

describe('<DayPickerInput />', () => {
  it('renders correctly default props', () => {
    const tree = renderer
      .create(<DayPickerInput input={{ name: 'name' }} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
