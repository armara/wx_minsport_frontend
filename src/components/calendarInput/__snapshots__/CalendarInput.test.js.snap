// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DayPickerInput /> renders correctly default props 1`] = `
<div
  className="DayPickerInput"
>
  <div
    className="calendar-input-field control has-icons-right"
  >
    <input
      autoComplete="off"
      className="form-input"
      name="name"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onPaste={[Function]}
      placeholder="дд.мм.гггг"
      value=""
    />
    <span
      className="icon is-small is-right"
    >
      <i
        className="ic-calendar"
      />
    </span>
  </div>
</div>
`;
